modules/users/server/config/strategies/twitter.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
 			var providerUserProfile = {
 				displayName: profile.displayName,
 				username: profile.username,
-				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value : undefined,
+				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value.replace('normal', 'bigger') : undefined,
```

Comment:
I think FB and Twitter would return a default image if the user hasn't uploaded one. I haven't tested this though. 

https://developers.facebook.com/docs/graph-api/reference/v2.2/user/picture

Looks like is_silhouette can be used to detect the if the default picture been replaced or not. 



> On Feb 1, 2015, at 11:18 AM, Ilan Biala <notifications@github.com> wrote:
> 
> In modules/users/server/config/strategies/twitter.js:
> 
> > @@ -26,7 +26,7 @@ module.exports = function(config) {
> >  			var providerUserProfile = {
> >  				displayName: profile.displayName,
> >  				username: profile.username,
> > -				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value : undefined,
> > +				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value.replace('normal', 'bigger') : undefined,
> Is there a case where a user won't have a photo that you have to check that it exists first? I would think even the bird would be the first profile picture, no? And is [0] the latest or the first?
> 
> —
> Reply to this email directly or view it on GitHub.
>